src/android/toga_android/window.py

```diff
@@ -41,7 +43,18 @@ def create_toolbar(self):
         pass
 
     def show(self):
-        pass
+        if not self.widget:
+            print("[Android] Showing an empty window is not supported.")
+            return
```

Comment:
We don't do this on other platforms - but we possibly should. At present, the app will fail with a weird "style is none" error on most platforms if you don't set content on the window.